confmap/provider/googlesecretsprovider/metadata.yaml

```
@@ -0,0 +1,8 @@
+type: googlesecretsprovider
+
+status:
+  class: provider
+  stability:
+    development: [provider]
+  codeowners:
+    active: [dashpole]
```

We will work on getting you membership to the org so you can be added as a codeowner here.

```suggestion
    active: [braydonk]
```